doo-swagger.json

```diff
@@ -5925,6 +6206,259 @@
         "data",
         "_links"
       ]
+    },
+    "WebhookCollectionPostRequest": {
+      "type": "object",
+      "properties": {
+        "user": {
+          "type": "string",
```

Comment:
In case the field has `string` type, please also specify `"minLength": 1` if the string can't contain empty value (in most cases)

Use this rule for all the fields with `string` type